index.html

```diff
@@ -205,7 +205,7 @@
     <section id="conformance">
       <p>
         This specification defines conformance criteria that apply to a single product: the
-        <dfn data-lt="user agent|user agents">user agent</dfn> that implements the interfaces that
+        <dfn data-lt="agents|user agents">user agent</dfn> that implements the interfaces that
```

Comment:
note for data-lt's, you don't need to repeat the defined term - so I removed those.